tracee-ebpf/main.go

```diff
@@ -304,6 +305,11 @@ func main() {
 				Usage:       "when to build the bpf program. possible options: 'never'/'always'/'if-needed'",
 				Destination: &buildPolicy,
 			},
+			&cli.BoolFlag{
+				Name:  "sort-events",
+				Value: false,
```

Comment:
why false by default?

Reply:
So I thought it might be good at start to make it optional.
However, I think you are right and most users probably will want it.
I will change it.

Reply:
do we need this for the current open sourced signatures?

Reply:
We partly need it.
The "stdio_over_socket" signature might be unpredictable (will probably miss memory freeing part of the times because of unordered events).